core-concepts/3_commands.md

```diff
@@ -199,11 +199,6 @@ from action commands.
 ~)
 </code></pre>
 
-#### Keywords
```

Comment:
Let us remember to add this section back in later. As of PennController 1.9, I think `self` is the only keyword